jest.config.js

```diff
@@ -0,0 +1,5 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
```

Comment:
Is this commented out code needed?

Reply:
I guess not. I can't remember why it's there but I've taken it out and it still works so that's good.